doc/developer-guides/hld/hv-partitionmode.rst

```diff
@@ -0,0 +1,358 @@
+
+A default E820 is used for all the guests in partition mode. This table
+shows the reference E820 layout. Zero page is created with this
+e820 info for all the guests.
+
++------------------------+
+| RAM                    |
+|                        |
+| 0 - 0xEFFFFH           |
++------------------------+
+| RESERVED (MPTABLE)     |
+|                        |
+| 0xF0000H - 0x100000H   |
++------------------------+
+| RAM                    |
+|                        |
+| 0x100000H - LOWMEM     |
++------------------------+
+| RESERVED               |
++------------------------+
+| PCI HOLE               |
++------------------------+
+| RESERVED               |
++------------------------+
+
+Platform info - mptable
+=======================
+
+ACRN, in partition mode, uses mptable to convey platform info to each
+guest.  Using this platform information, number of cores used for each
+guest, and whether the guest needs devices with INTX, ACRN builds
+mptable and copies it to the guest memory. In partition mode, ACRN uses
+physical APIC IDs to pass to the guests.
+
+I/O - Virtual devices
+=====================
+
+Port I/O is supported for PCI device config space 0xcfc and 0xcf8, vUART
+0x3f8, vRTC 0x70 and 0x71, and vPIC ranges 0x20/21, 0xa0/a1, and
+0x4d0/4d1.  MMIO is supported for vIOAPIC. ACRN exposes a virtual
+host-bridge at B.D:F 0.0:0 to each guest. Access to 256 bytes of config
+space for virtual host bridge is emulated.
+
+I/O - Pass-thru devices
+=======================
+
+ACRN, in partition mode, supports passing thru PCI devices on the
+platform. All the pass-thru devices are exposed as child devices under
+the virtual host bridge. ACRN does not support either passing thru
+bridges or emulating virtual bridges. Pass-thru devices should be
+statically allocated to each guest using the guest configuration. ACRN
+expects the developer to provide the virtual Bus.Device:Function (BDF)
+to BDF of the physical device mapping for all the pass-thru devices as
+part of each guest configuration.
+
+Run-time ACRN support for guests
+********************************
+
+ACRN, in partition mode, supports an option to pass-thru LAPIC of the
+physical CPUs to the guest. ACRN expects developers to specify if the
+guest needs LAPIC pass-thru using guest configuration. When guest
+configures vLAPIC as x2APIC, and if the guest configuration has LAPIC
+pass-thru enabled, ACRN passes the LAPIC to the guest. Guest can access
+the LAPIC hardware directly without hypervisor interception. During
+runtime of the guest, this option differentiates how ACRN supports
+inter-processor interrupt handling and device interrupt handling. This
+will be discussed in detail in the corresponding sections.
+
+.. figure:: images/partition-image16.png
+   :align: center
+
+
+Guest SMP boot flow
+===================
+
+The core APIC IDs are reported to the guest using mptable info. SMP boot
+flow is similar to sharing mode. Refer to <hld> for guest SMP
```

Comment:
Please replace `<hld>` with a link to the actual section in the original HLD.

Reply:
I agree.  @sainath14 Where in the HLD should this link?